core/arch/arm/plat-k3/drivers/ti_sci_transport.h

```diff
@@ -0,0 +1,74 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Texas Instruments SCI Transport Protocol Header
+ *
+ * Copyright (C) 2018-2025 Texas Instruments Incorporated - https://www.ti.com/
+ */
+
+#ifndef TI_SCI_TRANSPORT_H
+#define TI_SCI_TRANSPORT_H
+#include <stddef.h>
+#include <stdint.h>
+#include <tee_api_types.h>
+
+/**
+ * struct ti_sci_msg - Secure proxy message structure
+ * @len: Length of data in the Buffer
+ * @buf: Buffer pointer
+ *
+ * This is the structure for data used in ti_sci_transport_{send,recv}()
+ */
+struct ti_sci_msg {
+	size_t len;
+	uint8_t *buf;
+};
+
+/**
+ * mailbox_init() - Initialize the mailbox MMIO and SRAM regions
+ *
+ * This function initializes the mailbox MMIO and SRAM regions used for
+ * sending and receiving messages.
+ */
+void mailbox_init(void);
```

Comment:
This is a mailbox specific function. Is this needed to be declared here?

If not, remove it.